src/Microsoft.DotNet.ImageBuilder/src/Commands/BuildCommand.cs

```diff
@@ -298,6 +298,9 @@ private void PullBaseImages()
             .Where(repoData => repoData.Images != null)
             .SelectMany(repoData => repoData.Images.Values);
 
+        private IEnumerable<string> GetBuildTags() =>
```

Comment:
Naming pattern inconsistency with `GetBuiltImages`
```suggestion
        private IEnumerable<string> GetBuiltTags() =>
```